src/startupChecks.ts

```diff
@@ -1,4 +1,4 @@
-const requiredEnvVariables = ["FOUNDRY_MFE_HOST"];
+const requiredEnvVariables = ["FOUNDRY_MFE_HOST", "FOUNDRY_ENV"];
```

Comment:
This is to ensure we never are in a situation where FOUNDRY_ENV isn't set as an environment variable. Our code doesn't know what to do without a FOUNDRY_ENV, so that startupChecks function ensures we have all necessary env variables before trying to start up